### PR TITLE
Migrate lws go version to 1.22

### PR DESCRIPTION
**Pull request by liurupeng**

### CODE REVIEW

**Comment by ahg-g** [comment]
@liurupeng add yourself to https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes-sigs/lws/OWNERS

File: config/jobs/kubernetes-sigs/lws/lws-presubmit.yaml [hunk 1]
```diff
@@ -41,7 +41,7 @@ presubmits:
       description: "Run lws integration tests"
     spec:
       containers:
-      - image: public.ecr.aws/docker/library/golang:1.21
+      - image: public.ecr.aws/docker/library/golang:1.22
```
Comment (kannon92) [inline config/jobs/kubernetes-sigs/lws/lws-presubmit.yaml:44]: I would expect that the unit tests would also need this.

Why are we using kubekins for our unit tests?

**Comment by kannon92** [comment]
@ahg-g you will probably need to give an approval again.

**Pull request merged**